ERROR:django.request:Internal Server Error: /remove/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 111, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/home/animasola/Documents/TestCoffeecups/testingproject/testapp/views.py", line 75, in remove_log_object
    log_instance = ReqsHistory.objects.get(pk=int(request_id)).delete()
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/manager.py", line 131, in get
    return self.get_query_set().get(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/django/db/models/query.py", line 366, in get
    % self.model._meta.object_name)
DoesNotExist: ReqsHistory matching query does not exist.
